tensorboard/data/provider.py

```diff
@@ -391,6 +396,24 @@ def list_hyperparameters(self, ctx=None, *, experiment_ids):
         """
         pass
 
+    def read_hyperparameters(self, ctx=None, *, experiment_ids):
+        """Read hyperparameter values.
+
+        Args:
+          ctx: A TensorBoard `RequestContext` value.
+          experiment_ids: A Collection[string] of IDs of the enclosing
+            experiments.
+
+        Returns:
+          A Sequence[HyperparameterSessionGroup] describing the groups and their
```

Comment:
From offline discussion: Change to Collection[]. We don't plan to support ordering in this layer just yet.